common/src/main/resources/assets/wynntils/lang/zh_tw.json

```diff
@@ -1,982 +1,984 @@
+  "feature.wynntils.chatRedirect.scrollTeleport.description": "該如何顯示關於傳送失 (使用卷軸) 的訊息?",
+  "feature.wynntils.chatRedirect.scrollTeleport.name": "卷軸傳送訊息過濾",
+  "feature.wynntils.chatRedirect.shaman.description": "該如何顯示薩滿的特定訊息?",
   "feature.wynntils.chatRedirect.shaman.name": "薩滿訊息過濾",
-  "feature.wynntils.chatRedirect.soulPoint.description": "關於靈魂點訊息要如何顯示？",
-  "feature.wynntils.chatRedirect.soulPoint.name": "靈魂點訊息過濾",
-  "feature.wynntils.chatRedirect.speed.description": "弓箭手的加速效果訊息要如何顯示？",
-  "feature.wynntils.chatRedirect.speed.name": "加速教過訊息過濾",
-  "feature.wynntils.chatRedirect.toolDurability.description": "采集工具耐久低訊息要如何顯示？",
-  "feature.wynntils.chatRedirect.toolDurability.name": "采集工具耐久過濾",
-  "feature.wynntils.chatRedirect.unusedPoints.description": "未使用技能或屬性點要如何顯示 ？",
-  "feature.wynntils.chatRedirect.unusedPoints.name": "未使用技能或屬性點訊息過濾",
-  "feature.wynntils.chatTabs.name": "聊天頻道",
-  "feature.wynntils.chatTimestamp.formatPattern.description": "聊天室時間軸應該遵守哪版格式 ？",
-  "feature.wynntils.chatTimestamp.formatPattern.name": "聊天室時間軸的格式",
-  "feature.wynntils.chatTimestamp.invalidFormatMsg": "未有好的時間軸格式！直到您修，時間軸會被禁止",
-  "feature.wynntils.chatTimestamp.name": "聊天室時間軸",
-  "feature.wynntils.combatXpGainMessage.description": "您想把經驗附加放進疊加層嗎？",
-  "feature.wynntils.combatXpGainMessage.name": "戰鬥經驗附加訊",
-  "feature.wynntils.combatXpGainMessage.secondDelay.description": "戰鬥經驗訊息應該停留幾些秒？",
-  "feature.wynntils.combatXpGainMessage.secondDelay.name": "戰鬥經驗訊息到期時",
-  "feature.wynntils.commandAliases.name": "命令替代",
-  "feature.wynntils.commands.name": "Wynntils 命令",
-  "feature.wynntils.connectionProgress.name": "登錄Wynncraft的進步",
-  "feature.wynntils.containerSearch.filterInBank.description": "是否在銀行內加一位搜索欄？",
-  "feature.wynntils.containerSearch.filterInBank.name": "銀行內的搜索欄",
-  "feature.wynntils.containerSearch.filterInGuildBank.description": "是否在公會銀行內加一位搜索欄？",
-  "feature.wynntils.containerSearch.filterInGuildBank.name": "公會銀行內的搜索欄",
-  "feature.wynntils.containerSearch.filterInGuildMemberList.description": "是否在公會的成員榜內加一位搜索欄？",
-  "feature.wynntils.containerSearch.filterInGuildMemberList.name": "公會的成員榜內的搜索欄",
-  "feature.wynntils.containerSearch.highlightColor.description": "搜索到的物品加的強調應該渲染什麼顏色？",
-  "feature.wynntils.containerSearch.highlightColor.name": "強調色",
-  "feature.wynntils.containerSearch.name": "箱子搜索到的加強調",
-  "feature.wynntils.cosmeticsPreview.name": "化妝裝箱預覽",
-  "feature.wynntils.customBarsOverlay.name": "特別的叠加條",
+  "feature.wynntils.chatRedirect.soulPoint.description": "該如何顯示靈魂點數的訊息?",
+  "feature.wynntils.chatRedirect.soulPoint.name": "靈魂點數訊息過濾",
+  "feature.wynntils.chatRedirect.speed.description": "該如何顯示速度效果的訊息?",
+  "feature.wynntils.chatRedirect.speed.name": "速度效果訊息過濾",
+  "feature.wynntils.chatRedirect.toolDurability.description": "該如何顯示採集工具耐久度的訊息?",
+  "feature.wynntils.chatRedirect.toolDurability.name": "採集工具耐久度過濾",
+  "feature.wynntils.chatRedirect.unusedPoints.description": "該如何顯示關於未使用能力點或技能點的訊息?",
```

Comment:
I had `屬性點` as precedent for skill point, but this might also be fine. I am a bit concerned about using `能力` for both skill points and ability tree earlier though

Reply:
You are right, I'll change this into `屬性點`, thanks for review.